Code/Framework/AzCore/AzCore/Math/Vector2.cpp

```diff
@@ -235,7 +235,7 @@ namespace AZ
                 Method("GetElement", &Vector2::GetElement)->
                 Method("SetElement", &Vector2::SetElement, { { {"Index", ""}, {"Value",""} } })->
                     Attribute(Script::Attributes::ExcludeFrom, Script::Attributes::ExcludeFlags::All)->
-                Method("GetLength", &Vector2::GetLength)->
+                Method("GetLength", &Vector2::GetLength, { "Source", "The source value to calculate its magnitude." }, {})->
```

Comment:
The source of the magnitude calculation.

Reply:
done